pages/rules.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/style.css">
+    <title>Quizmee | Rules</title>
```

Comment:
Interchange the order of naming here, the title of the page should come before the App name, because on smaller tabs, only the first name would be visible

Reply:
okay

Reply:
done